packages/as-transformer/src/transformers/massaExport.ts

```diff
@@ -146,7 +146,7 @@ export class MassaExport {
       wrapper +=
         '  generateEvent(' +
         `\`Result${this.functionName}:` +
-        "'${response.buffer.toString()}'`);\n";
+        "'${new Uint8Array(response.buffer).toString()}'`);\n";
```

Comment:
```suggestion
        "'${encode(response)}'`);\n";
```
with the following import:
```typescript
import { encode } from "as-base64";
```